ironic/drivers/modules/drac/raid.py

```diff
@@ -769,6 +769,35 @@ def delete_configuration(self, task):
 
         return _commit_to_controllers(node, list(controllers))
 
+    @METRICS.timer('DracRAID.reset_idrac')
```

Comment:
I think these cleaning steps should be in the DracManagement class instead of the RAID class.

Reply:
I have added cleaning steps in the DracManagement class in latest commit, can you please review it.

Reply:
Can you back out the changes to this file?  This file is still showing up as modified.

Reply:
Done